src/api/types/generated-schema.ts

```diff
@@ -103,10 +103,6 @@ export interface paths {
     /** Gets all species associated to a participant project. */
     get: operations["getSpeciesForProject"];
   };
-  "/api/v1/accelerator/projects/{projectId}/species/snapshots/{deliverableId}": {
```

Comment:
@nickgraz this endpoint is deleted in the backend and I'm getting an error in the build because it is used in the FE. Was the removal intentional? Are we missing removing something in the FE?

Reply:
It hasn't been merged in yet, sorry. It should be merged in today